cli/src/template.rs

```diff
@@ -48,11 +48,12 @@ export const IDL: {} = {};
 
 pub fn cargo_toml(name: &str) -> String {
     format!(
-        r#"[package]
+        r#"cargo-features = ["edition2021"]
```

Comment:
Is this can be a replacement of field `rust-version` in Cargo.toml?

Reply:
yes, and it's either way necessary for edition=2021

Reply:
I don't think so, I'm only getting the above error when running an older cargo version. This feature should not be necessary.

Reply:
at 
cargo 1.57.0 (b2e52d7ca 2021-10-21)
rustc 1.57.0 (f1edd0429 2021-11-29)
a.k.a latest version i'm still having the error above.